### PR TITLE
Cookbook cleanup: Relocating 'Install on macOS or Windows'

### PR DESCRIPTION
**Pull request by michalmilc90**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | n/a
| Versions      | all

Relocating the 'Install on macOS or Windows' section.

### CODE REVIEW

**Comment by michalmilc90** [comment]
Needs relocating to other location in the doc.

File: docs/community_resources/installing-on-mac-os-and-windows.md [hunk 1]
```diff
@@ -1,50 +1,55 @@
-# Installing eZ Platform on macOS or Windows
+# Install eZ Platform on macOS or Windows
 
-Installing eZ Platform for production is only supported on Linux.
-See [Install eZ Platform](../getting_started/install_ez_platform.md) for a regular installation guide.
+This page show how to perform a installation of eZ Platform on macOS or Windows.
```
Comment (DominikaK) [inline docs/community_resources/installing-on-mac-os-and-windows.md]:
```suggestion
This page shows how to perform an installation of eZ Platform on macOS or Windows.
```

Comment (DominikaK) [inline docs/community_resources/installing-on-mac-os-and-windows.md]:
```suggestion
This page explains how to install eZ Platform on macOS or Windows.
```

File: docs/community_resources/installing-on-mac-os-and-windows.md [hunk 2]
```diff
@@ -1,50 +1,55 @@
...
+ 
+- For Windows: Apache
+- For macOS: Apache/nginx
+
 The instructions below assumes you are using Apache.
 
 ??? "Windows"
 
-    Locate `php.ini` file and open it in a text editor. Provide missing values to relevant parameters e.g. `date.timezone` and `memory_limit`:
+    Locate `php.ini` file and open it in a text editor.
```
Comment (DominikaK) [inline docs/community_resources/installing-on-mac-os-and-windows.md]:
```suggestion
    Locate the `php.ini` file and open it in a text editor.
```

File: docs/community_resources/installing-on-mac-os-and-windows.md [hunk 3]
```diff
@@ -71,7 +76,7 @@ The instructions below assumes you are using Apache.
 Download and extract an archive into the location where you want your project root directory to be from [ezplatform.com](https://ezplatform.com/#download-option) (for open-source version) or from the [Support portal](https://support.ez.no/Downloads) (for eZ Enterprise), or clone the [GitHub repository](https://github.com/ezsystems/ezplatform):
 
 ``` bash
-git clone https://github.com/ezsystems/ezplatform .
+git clone https://github.com/ezsystems/ezplatform
```
Comment (DominikaK) [inline docs/community_resources/installing-on-mac-os-and-windows.md]: I think the full stop should remain here, to remain consistent with the main install guide.

File: docs/community_resources/installing-on-mac-os-and-windows.md [hunk 4]
```diff
@@ -123,21 +128,17 @@ The configuration requires providing the following parameters:
...
 
-Before executing the following command make sure that the user set during `composer install` has sufficient permissions.
-
-Install eZ Platform with:
+Before executing the following command, ensure that the user set during `composer install` has sufficient permissions.
 
-``` bash
-composer ezplatform-install
-```
+Install eZ Platform by running: `composer ezplatform-install`.
```
Comment (DominikaK) [inline docs/community_resources/installing-on-mac-os-and-windows.md]: I'd leave the command in a separate code block. True, it is only one line, but it's one of the two main commands on this page and should be emphasised.

**Pull request merged**